### PR TITLE
Mjm delivery tracker

### PR DESCRIPTION
**Pull request by m-nt-s-no**
@DPI-WE/technical-associates, please review

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 require "active_support/core_ext/integer/time"
 
 Rails.application.configure do
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (aldoportillo) [inline config/environments/development.rb:4]: why did you configure action mailer?

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+
+  # CREATE
+  post("/insert_package", { :controller => "packages", :action => "create" })
+          
+  # READ
+  get("/packages", { :controller => "packages", :action => "index" })
+  
+  # UPDATE
+  
+  post("/modify_package/:path_id", { :controller => "packages", :action => "update" })
```
Comment (aldoportillo) [inline config/routes.rb:14]: I like how you are descriptive with your routes but if this was used as an api it would be difficult to track the endpoints so keeping them restful will be your best friend. 

TLDR: You can have multiple routes with the same name as long as the http verbs are different: 

get('/packages', ...)
post('/packages', ...)

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -34,12 +34,6 @@
 end
 
 RSpec.configure do |config|
-  # Suppress RSpec warnings about potential false positives
```
Comment (aldoportillo) [inline spec/rails_helper.rb:37]: for why?

**Review by aldoportillo — APPROVED** [review]
I made some notes regarding some good practices but overall good.

File: config/environments/development.rb (hunk 1 above)
Comment (m-nt-s-no) [reply config/environments/development.rb:4]: only because github told me to

File: config/routes.rb (hunk 2 above)
Comment (m-nt-s-no) [reply config/routes.rb:14]: Yes! Learned about that right after I submitted

File: spec/rails_helper.rb (hunk 3 above)
Comment (m-nt-s-no) [reply spec/rails_helper.rb:37]: I'll be honest...I don't remember this.
